src/server/verification/__tests__/verificationAPI.js

```diff
@@ -93,6 +102,37 @@ describe('verificationAPI', () => {
     expect(dbUser.emailVerificationCode).toBeTruthy()
   })
 
+  test('/verify/sendemail should fail with 429 status - too many requests (rate limiter)', async () => {
+    const token = await getToken(server)
+
+    await storage.model.deleteMany({ fullName: new RegExp('test_user_sendemail', 'i') })
+
+    const user = await UserDBPrivate.updateUser({
+      identifier: '0x7ac080f6607405705aed79675789701a48c76f55',
+      fullName: 'test_user_sendemail'
+    })
+
+    expect(user).toBeTruthy()
+
+    // the 4th request should fail with status 429
+    // we see only 3 requests executed from this test. But in previous test we executing one more api call to the same endpoint.
+    // so in summary we have 4 requests
+    for (let i = 1; i <= 3; i++) {
```

Comment:
loop until you get 429 dont use fixed number

Reply:
done. see new commits.